<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultEvent</class>
 <widget class="QWidget" name="DefaultEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>401</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>Time - Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="summaryLabel">
      <property name="text">
       <string>Summary</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
